test/framework/src/main/java/org/elasticsearch/index/mapper/NumberFieldMapperTests.java

```diff
@@ -315,7 +317,7 @@ public void testMetricAndDocvalues() {
     }
 
     @Override
-    protected final Object generateRandomInputValue(MappedFieldType ft) {
+    protected Object generateRandomInputValue(MappedFieldType ft) {
```

Comment:
Needs to be made non-final so `UnsignedLongFieldMapperTests` and `ScaledFloatFieldMapperTests` can override it